recipes/rnabob/removeman.patch

```diff
@@ -0,0 +1,10 @@
+--- Makefile	2016-09-18 01:24:06.664551971 +0200
++++ Makefile-new	2016-09-18 02:00:43.872926057 +0200
+@@ -58,7 +58,6 @@
+ 
+ install: $(PROGS)
+ 	cp $(PROGS)   $(BINDIR)/
+-	cp rnabob.man $(MANDIR)/man1/rnabob.1
```

Comment:
The problem is that ```$HOME``` is hardcoded in the Makefile.
I tried to make it more generic in the last commit by setting ```HOME=$PREFIX```. Otherwise, I can rewrite the Makefile through a patch.
But, I still need the patch because the man folder doesn't exist in the conda build environment in the bioconda docker.